### PR TITLE
Fixed issue #1100

### PR DESCRIPTION
**Pull request by Rishabhk07**
Signed-off-by: Rishabh Khanna <rishabhkhanna777@gmail.com>

### CODE REVIEW

**Comment by Princu7** [comment]
@Rishabhk07 Please give a link to the live gh-pages :)

**Comment by geekyd** [comment]
@Rishabhk07 Great work! Please provide live link to the app.

**Comment by Rishabhk07** [comment]
https://rishabhk07.github.io/open-gh/tracks.html 
i had edit the FossAsia16 zip for 2 speakers on same track, it can be seen in Grand Opening track

File: src/backend/templates/tracks.hbs [hunk 1]
```diff
@@ -118,6 +118,7 @@
 		             	              {{#speakers_list}}
 		             	                {{name}}
 		             	                {{#if organisation}}({{organisation}}){{/if}}
+                                          {{#if @last}}{{else}},{{/if}}
```
Comment (aayusharora) [inline src/backend/templates/tracks.hbs]: @Rishabhk07 Please remove a space after `else`

Comment (Rishabhk07) [inline src/backend/templates/tracks.hbs]: @aayusharora  space between first speaker and comma ?
like this https://rishabhk07.github.io/open-gh/tracks.html

**Comment by Rishabhk07** [comment]
@aayusharora i have removed the space and formatted the code, please review :)

**Pull request merged**